graphiql/index.html

```diff
@@ -22,7 +22,8 @@
     <div id="workspace" class="graphiql-workspace"></div>
 
     <script>
-      var config = new graphiqlWorkspace.AppConfig("GraphiQL", {});
+      var host = window.location.host
+      var config = new graphiqlWorkspace.AppConfig("GraphiQL", {defaultUrl: "http://"+host+"/query"});
```

Comment:
Use `window.location.protocol` for protocol, if user would enter this page on HTTPS protocol, it would result in *Not safe* in their url bar. Also, spaces surrounding operators, please.